src/data-tutorials/users.tsx

```diff
@@ -1119,6 +1119,13 @@ const Users: User[] = [
     website: 'blog/tutorials/Understanding-the-Fundamentals-of-Auction-Programming-in-Solidity',
     tags: ['celo', 'intermediate', 'celosage'],
   },
+  { 
+    title: 'Understanding the Role of Proposals and Referendums in Celo Governance',
+    description: 'A Comprehensive Breakdown of the Role of Proposals and Referendums in Celo Governance',
+    preview: require('./showcase/intermediate/Understanding-the-Role-of-Proposals-and-Referendums-in-Celo-Governance.png'),
```

Comment:
Change this name of the file "Understanding-the-Role-of-Proposals-and-Referendums-in-Celo-Governance.png" to "celo-governance.png"